lib/facter/self_service.rb

```diff
@@ -227,6 +227,15 @@
     { S0031: no_old_packages }
   end
 
+  chunk(:S0032) do
+    # MCollective/ActiveMQ enabled/exists?
+    # rubocop:disable Metrics/LineLength
+    mco_check = PuppetSelfService.service_running_enabled('mcollective') || PuppetSelfService.service_file_exist?('mcollective') || PuppetSelfService.get_resource('cron', 'pe-mcollective-metadata').nil?
+    amq_check = PuppetSelfService.service_running_enabled('pe-activemq') || PuppetSelfService.service_file_exist?('pe-activemq')
+    { S0032: !mco_check || !amq_check }
```

Comment:
I think the logic here should be that both the `mco_check` and `amq_check` are not true.
```suggestion
    { S0032: !mco_check && !amq_check }
```